lib/pghero/version.rb

```diff
@@ -1,3 +1,3 @@
 module PgHero
-  VERSION = "2.2.0"
+  VERSION = "2.2.1"
```

Comment:
It's generally a good open source practice to avoid updating the version. Maintainers typically have their own process for this.